syntax = "proto3";
package reco;

option java_multiple_files = true;
option java_package = "co.edu.unal.campus.protos";
option java_outer_classname = "RecommendationProto";


service Recommendation {
  rpc Recommend (RecommendRequest) returns (RecommendResponse);
}

message RecommendRequest {
  int32 user_id = 1;
  int32 limit = 2;
}

message RecommendResponse {
  repeated int32 event_ids = 1;
}